addons/website/models/website_menu.py

```diff
@@ -186,7 +205,7 @@ def _is_active(self):
           impossible to compare anyway as the client is not sending the anchor
           to the server as per RFC)
         - query string parameters should be the same to be considered equal, as
-          those could drasticaly alter a page result
+          those could drastically alter a page result
```

Comment:
I always tend to keep those changes separated :) You can revert, we may change it another time, when modifying that part of the code.